src/widgetresize.js

```diff
@@ -187,6 +169,43 @@ export default class WidgetResize extends Plugin {
 	_getResizerByViewElement( viewElement ) {
 		return this._resizers.get( viewElement );
 	}
+
+	/**
+	 * @private
```

Comment:
Private methods are accessible only inside the class. If you use them for tests they should be protected.